invoker/rh-changes/pom.xml

```diff
@@ -82,6 +82,12 @@
                 <version>${redhat.che.version}</version>
                 <type>war</type>
             </dependency>
+            <dependency>
+                <groupId>com.redhat.che</groupId>
+                <artifactId>fabric8-ide-stacks</artifactId>
+                <version>${redhat.che.version}</version>
```

Comment:
since it's a customization of upstream che, I think using `${redhat.che.version}` is good.